cmake/opendds_target_sources.cmake

```
@@ -216,7 +216,7 @@ function(opendds_export_header target)
     return()
   endif()
 
-  # Need come before configure_file
+  # Need to be defined before configure_file
```

```suggestion
  # Needs to be defined before configure_file
```
? not entirely sure what this is referring to

---

`uppercase_target` and `export_macro` are used in the export template. I moved them after `configure_file` on my machine and was confused why my builds where breaking like they were completely missing the export headers.